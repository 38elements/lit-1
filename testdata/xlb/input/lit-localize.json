{
  "sourceLocale": "en",
  "targetLocales": ["es-419"],
  "tsConfig": "tsconfig.json",
  "tsOut": "tsout",
  "xlbDir": "xlb"
}
