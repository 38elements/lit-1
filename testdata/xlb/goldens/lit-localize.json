{
  "$schema": "https://raw.githubusercontent.com/PolymerLabs/lit-localize/master/config.schema.json",
  "sourceLocale": "en",
  "targetLocales": [
    "es-419"
  ],
  "tsConfig": "tsconfig.json",
  "tsOut": "tsout",
  "xlbDir": "xlb"
}
